scaleway/helpers_iot.go

```diff
@@ -12,6 +16,8 @@ import (
 const (
 	defaultIoTRetryInterval = 5 * time.Second
 	defaultIoTHubTimeout    = 5 * time.Minute
+	mqttCaNameFile          = "iot-hub-ca.pem"
+	mqttCaURLDownload       = "https://iot.s3.nl-ams.scw.cloud/certificates/"
```

Comment:
Is it a problem that region is hard coded here? Would it be the same CA regardless of the region?

Reply:
I check with Julien the certificat is in a s3 in ams and it won't change in the futur, so should not be an issue if it is hard coded